docs/source/conf.py

```diff
@@ -70,25 +70,39 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
-html_theme_path = [alabaster.get_path()]
+html_theme = 'sphinx_rtd_theme'
 
-# Custom sidebar templates, must be a dictionary that maps document names
-# to template names.
-#
-# This is required for the alabaster theme
-# refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
 html_sidebars = {
     '**': [
-        'about.html',
-        'navigation.html',
+        'localtoc.html',
         'relations.html',
         'searchbox.html',
     ]
 }
-html_theme_options = {
-    'fixed_sidebar': True,
 
+# Custom sidebar templates, must be a dictionary that maps document names
+# to template names.
+#
+# This is required for the alabaster theme
+# refs: http://alabaster.readthedocs.io/en/latest/installation.html#sidebars
+#html_sidebars = {
+#    '**': [
+#        'about.html',
+#        'navigation.html',
+#        'relations.html',
+#        'searchbox.html',
+#    ]
+#}
```

Comment:
If the theme is changed do you still need this configs for the previous theme?

Reply:
In the future I hope to revive a better-looking theme but you're right, in theory we can delete the contents (but better leave the html_sidebars stay there - it is a general setting used in different themes.

Reply:
@LiraLemur this code will be available in git history - so do not need to comment it. Delete and look at history when needed.

Reply:
I can look at it but those comments were a great help when I started doing that configuration stuff long time ago. But ok, it's not a big deal :)